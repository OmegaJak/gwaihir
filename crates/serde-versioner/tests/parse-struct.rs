use serde::{Deserialize, Serialize};
use serde_versioner::version;

#[derive(Serialize, Deserialize)]
#[version(crate::ThingyV1)]
struct Thingy {
    a: i32,
    b: u64,
}

fn main() {}
