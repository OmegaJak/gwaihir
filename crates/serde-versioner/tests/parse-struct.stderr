error: cannot find attribute `serds` in this scope
 --> tests/parse-struct.rs:5:1
  |
5 | #[version]
  | ^^^^^^^^^^ help: a derive helper attribute with a similar name exists: `serde`
  |
  = note: this error originates in the attribute macro `version` (in Nightly builds, run with -Z macro-backtrace for more info)
